src/action-ros-ci.ts

```diff
@@ -183,10 +183,17 @@ async function run() {
 			options
 		);
 
+		// Let Eloquent be the default distro used for rosdep
+		let rosdepRosdistro = "--rosdistro eloquent";
```

Comment:
One more thing: Can we keep `--rosdistro` as part of the bash command?
That way the naming is isolated from the flag (in case it needs to be changed sometime in the future).

Reply:
Sure. This made me notice another thing: since the action supports sourcing multiple binary installations (e.g. melodic and dashing) it's not clear to me what distro we should use to install dependencies for. Naively, we could run a rosdep install command for each distro installed (I'm not sure if there are any undesired consequences to this). If this does cause issues, then we could introduce another input to the action for choosing which distros to install dependencies from.

Reply:
Given the order of the `source` commands, I'd expect dependencies installed in the latter rosdistro to be used anyways (ie. the last one specified in `sourceRosBinaryInstallation`). So I've changed the code to install dependencies for the latter rosdistro: fc89dc4